src/libtest/lib.rs

```diff
@@ -1374,54 +1462,32 @@ pub fn run_test(
     opts: &TestOpts,
     force_ignore: bool,
     test: TestDescAndFn,
+    strategy: RunStrategy,
     monitor_ch: Sender<MonitorMsg>,
     concurrency: Concurrent,
 ) {
     let TestDescAndFn { desc, testfn } = test;
 
-    let ignore_because_panic_abort = cfg!(target_arch = "wasm32")
-        && !cfg!(target_os = "emscripten")
-        && desc.should_panic != ShouldPanic::No;
```

Comment:
This arguably may want to stay although perhaps because of a different reason. These platforms don't have process support either, so I think this may cause tests to start failing?

(this is fine to sort out later, we don't have to deal with it up front just yet, just a small heads up)

Reply:
Good point, I put it back in.